go/cmd/dolt/commands/diff_output.go

```diff
@@ -247,6 +247,12 @@ func (t tabularDiffWriter) WriteSchemaDiff(ctx context.Context, toRoot *doltdb.R
 		cli.Println(textdiff.LineDiff(fromCreateStmt, toCreateStmt))
 	}
 
+	if len(td.FromFks) > 0 || len(td.ToFks) > 0 {
```

Comment:
This should be checking if the foreign keys have been resolved. AFAICT, this is going to print `resolved foreign key` whenever _any_ change is made to a foreign key. In fact, if a foreign key was deleted, we'll still get `resolved foreign key`. Need some tests for all of the scenarios:

- New foreign key is added and resolved
- New foreign key is added and not resolved
- Existing foreign key that was resolved is deleted
- Existing foreign key that was not resolved is deleted
- Existing foreign key that was resolved is "_modified_" (dropped then readded with a change)
- Existing foreign key that was not resolved is "_modified_"
- Existing foreign key is resolved